kytos/core/controller.py

```diff
@@ -25,7 +25,8 @@
 from kytos.core.api_server import APIServer
 from kytos.core.buffers import KytosBuffers
 from kytos.core.events import KytosEvent
-from kytos.core.helpers import now, start_logger
+from kytos.core.helpers import now
+from kytos.core.log_manager import LogManager
```

Comment:
My only concern is here. I think that is better if it was something, like:

```
from kytos.core.log import LogManager
```
Or

```
from kytos.core.log.manager import LogManager
```